### PR TITLE
Education-Translations (Turkish) Chapter IX - Testing a Trading Idea The Other Stages

### PR DESCRIPTION
**Pull request by tigers40**
Docs-Tutorials\T\Testing\Testing-a-Trading…\testing-a-trading-idea-the-other-stages-008-tutorial-step-the-target-size.json
Docs-Tutorials\T\Testing\Testing-a-Tra…\testing-a-trading-idea-the-other-stages-009-tutorial-step-the-whole-balance.json
Docs-Tutorials\T\Testing\Testi…\testing-a-trading-idea-the-other-stages-010-tutorial-step-initial-balance-vs.-balance.json
Docs-Tutorials\T\Testing\Testing-a-Tradin…\testing-a-trading-idea-the-other-stages-011-tutorial-step-its-the-balance.json
Docs-Tutorials\T\Testing\Testing-a-Trading-Ide…\testing-a-trading-idea-the-other-stages-012-tutorial-step-there-it-is.json

### CODE REVIEW

File: Projects/Foundations/Schemas/Docs-Tutorials/T/Testing/Testing-a-Trading-Idea-The-Other-Stages/testing-a-trading-idea-the-other-stages-041-tutorial-step-ready-to-rock..json [hunk 1]
```diff
@@ -50,13 +65,25 @@
...
+                    "language": "TR",
+                    "text": "Bollinger Verileri(Bollinger Data): \"Bollinger Bantları\".",
+                    "updated": 1641555841164
                 }
             ]
         },
         {
             "style": "List",
             "text": "Simulation Objects: \"Strategies\", \"Market Buy Orders\", \"Market Sell Orders\", \"Position Base Asset\", \"Stop Loss\", and \"Take Profit\".",
-            "updated": 1622549420151
```
Comment (gitforkgit) [inline Projects/Foundations/Schemas/Docs-Tutorials/T/Testing/Testing-a-Trading-Idea-The-Other-Stages/testing-a-trading-idea-the-other-stages-041-tutorial-step-ready-to-rock..json:59]: The thing added was a comma.

Comment (harrellbm) [reply Projects/Foundations/Schemas/Docs-Tutorials/T/Testing/Testing-a-Trading-Idea-The-Other-Stages/testing-a-trading-idea-the-other-stages-041-tutorial-step-ready-to-rock..json:59]: @gitforkgit Great job!

**Comment by harrellbm** [comment]
@gitforkgit @tigers40 Something is till getting flagged as a merge conflict.  I think the full vs code route is the way to go to figure this one out

**Comment by tigers40** [comment]
> @gitforkgit @tigers40 Birleştirme çakışması olarak işaretlenene kadar bir şeyler var. Bence tam vs kod rotası bunu çözmenin yolu

Thanks @gitforkgit. So what am I supposed to do at this stage?

**Pull request closed without merging**